app/components/shared/personal_statement_component.rb

```diff
@@ -14,5 +14,9 @@ def initialize(application_form:, editable: false)
 
 private
 
+  def header
+    FeatureFlag.active?(:one_personal_statement) && application_form.single_personal_statement? ? 'Personal statement' : I18n.t('personal_statement.vocation')
```

Comment:
Looks like "Personal statement" is already in the locale file, so could reference that here?

```suggestion
    if FeatureFlag.active?(:one_personal_statement) && application_form.single_personal_statement?    
      I18n.t('page_titles.personal_statement')
    else
      I18n.t('personal_statement.vocation')
    end
```

Reply:
👌